aicf-announces-indian-squad-for-44th-chess-olympiad.md added (#1688)
slurpcode/slurp: doc/source/aicf-announces-indian-squad-for-44th-chess-olympiad.md

# 🏯 AICF announces Indian Squad for 44th Chess Olympiad

> All India Chess Federation (AICF) on Monday announced the India's 20

Reviewer: ```suggestion
All India Chess Federation (AICF) on Monday announced India's 20
```

> member squad which is biggest ever Olympiad squad, for the 44th Chess

Reviewer: ```suggestion
member squad which is the biggest ever Olympiad squad, for the 44th Chess
```

Olympiad as India prepares to host the prestigious tournament of Chess
in Chennai, India. Olympiad will be held from 28th July to 10th August.

> <br>India being the host are eligible to field two teams each in Open

Reviewer: ```suggestion
<br>India being the host are eligible to field two teams each in the Open
```

category as well as women's section for the first time ever. Being able

> to play with two teams each in both field has certainly increased the

Reviewer: ```suggestion
to play with two teams each in both fields has certainly increased the
```

> chances of winning Medals for India. The event is biggest chess event

Reviewer: ```suggestion
chances of winning Medals for India. The event is the biggest chess event
```

> chances of winning Medals for India. The event is the biggest chess event

Reviewer: ```suggestion
chances of winning medals for India. The event is the biggest chess event
```

and likely to witness participation of whooping 150 countries and more.

> <br>India A team consist of GM Vidit Gujrathi, who led India in 2020 online

Reviewer: ```suggestion
<br>India A team consist of GM Vidit Gujrathi, who led India in the 2020 online
```

chess Olympiad to win Gold alongside GM Pentala Harikrishna the top
Indian, GM Krishnan Sasikiran, young star Arjun Erigaisi and S L
Narayanan in the open section.

> <br>Last one year was an impressive for Arjun Erigaisi winning and competing

Reviewer: ```suggestion
<br>The last one year was an impressive for Arjun Erigaisi winning and competing
```

> in top events and he will be the player to watch out as he makes debut

Reviewer: ```suggestion
in top events and he will be the player to watch out as he makes debut.
```

> in top events and he will be the player to watch out as he makes debut.

Reviewer: ```suggestion
in top events and he will be the player to watch out for as he makes debut.
```

> also it will be debut event for S L Narayanan.

Reviewer: ```suggestion
Also it will be debut event for S L Narayanan.
```

<br>![Team India A; credits: AICF India](_static/images/aicf-announces-indian-squad-for-44th-chess-olympiad/image1.jpg)

Team India A; credits: AICF India

<br>India B team consist of young Indian guns performing consistently who
are the topic of discussion in chess world, Nihal Sarin, R
Praggnanandhaa, D Gukesh and Raunak Sadhwani. All are under 20 and will

> be making their debut in this event. Along them will also have

Reviewer: ```suggestion
be making their debut in this event. Along with them we also have
```

experienced player Adhiban Baskaran who has also won the bronze in team
in 2014 chess Olympiad.

<br>![Team India B; credits: AICF India](_static/images/aicf-announces-indian-squad-for-44th-chess-olympiad/image2.jpg)

Team India B; credits: AICF India

<br> AICF Secretary Bharat Singh Chauhan said "Opportunity to field two teams
at the Chess Olympiad opens the door for many young Indian talents to

> showcase their game at the biggest stage which otherwise would have been

Reviewer: ```suggestion
showcase their game at the biggest stage which otherwise would only have been
```
